data.txt

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Is this a leftover data file from running the application? Try adding the filename to .gitignore so that this file won't be commited.